tests/test_ec_v2.py

```diff
@@ -4,7 +4,6 @@
 
 import pytest
 from beancount.core.data import Amount, Balance
-
```

Comment:
If you remove this change, the linter should stop complaining as well.

Reply:
A global `isort` 5.12 interfered with the local `isort` 5.13 where the difference seems to come from